### ISSUE TITLE
where to find th sequentially-bootstrapped versions !!!!!!

### ISSUE THREAD

**Issue by ashraf-01** [dmlc/xgboost#6046]
Hi all ,
am looking in to the possibility of applying the  of boosting gradient method within the RF algorithm in python , but mainly because of ordering of the sample points is important to my applications , mainly time series , am looking in to applying the sequentially -bootstrapped version of the above ?
basically I wanna still be able to find a probability on the predictions generated after the fitting step 
Your input is highly appreciated with examples included would be much more serving 

thanks

**Comment by hcho3** [comment]
XGBoost does not recognize the concept of temporal sequence. I suggest that you use neural networks to explicitly model temporal sequences. See discussion in https://discuss.xgboost.ai/t/forecasting-same-learner-to-two-copies-of-data-set/1842/2?u=hcho3

**Comment by ashraf-01** [comment]
alright
